pages/common/lerna.md

```diff
@@ -0,0 +1,24 @@
+# lerna
+
+> A tool for managing JavaScript projects with multiple packages.
+> More information: <https://lerna.js.org/>.
+
+- Initialize project files (`lerna.json`, `package.json`, `.git`, …):
```

Comment:
Don't use special chars like `…` could be not compatible with the user's terminal.
```suggestion
- Initialize project files (`lerna.json`, `package.json`, `.git`, .etc):
```

Reply:
@navarroaxel right, but it should be `etc.` not `.etc`:
```suggestion
- Initialize project files (`lerna.json`, `package.json`, `.git`, etc.):
```

Reply:
Why not `...`?

Reply:
because `...` is for a pattern, like the sequence `1, 2, ...`